modules/about-using-gpu-operator.adoc

```diff
@@ -1,21 +1,87 @@
+NVIDIA configuration::
+For the configuration method that NVIDIA supports, refer to link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/openshift-virtualization.html[NVIDIA GPU Operator with {VirtProductName}] in the NVIDIA documentation.
 
-For more information about using the NVIDIA GPU Operator to provision worker nodes for running GPU-accelerated VMs, see link:https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/openshift/openshift-virtualization.html[NVIDIA GPU Operator with OpenShift Virtualization].
+Red Hat configuration (recommended)::
+The link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/openshift-virtualization.html[NVIDIA documentation] is still the basis of the Red Hat-supported method, but it is important to note that Red Hat uses a different configuration when testing {VirtProductName}. In particular, several aspects of the `ClusterPolicy` configuration that NVIDIA recommends are incompatible with {VirtProductName}'s scheduling functionality.
++
+To use the configuration that Red Hat has validated to work with {VirtProductName}, diverge from the NVIDIA documentation in the following ways:
++
+* Do not set the `DisableMDEVConfiguration` feature gate.
+* Ensure that your `ClusterPolicy` configuration values match the following example:
++
+.Example manifest
+[source,yaml]
+----
+kind: ClusterPolicy
+apiVersion: nvidia.com/v1
+metadata:
+  name: gpu-cluster-policy
+spec:
+  operator:
+    defaultRuntime: crio
+    use_ocp_driver_toolkit: true
+    initContainer: {}
+  sandboxWorkloads:
+    enabled: true
+    defaultWorkload: vm-vgpu
+  driver:
+    enabled: false <1>
+  dcgmExporter: {}
+  dcgm:
+    enabled: true
+  daemonsets: {}
+  devicePlugin: {}
+  gfd: {}
+  migManager:
+    enabled: true
+  nodeStatusExporter:
+    enabled: true
+  mig:
+    strategy: single
+  toolkit:
+    enabled: true
+  validator:
+    plugin:
+      env:
+        - name: WITH_WORKLOAD
+          value: "true"
+  vgpuManager:
+    enabled: true <2>
+    repository: <vgpu_container_registry> <3>
+    image: <vgpu_image_name> <4>
+    version: nvidia-vgpu-manager
+  vgpuDeviceManager:
+    enabled: false <5>
+    config:
+      name: vgpu-devices-config
+      default: default
+  sandboxDevicePlugin:
+    enabled: false
+  vfioManager:
+    enabled: false <6>
+----
+<1> Set this value to `false`. Not required for VMs.
+<2> Set this value to `true`. Required for using vGPUs with VMs.
+<3> Substitute `<vgpu_container_registry>` with your registry value.
+<4> This image should be created once for each RHCOS X.Y version. 
+// TODO: check the meaning of this and rephrase
+<5> Set this value to `false` to allow {VirtProductName} to configure mediated devices instead of the NVIDIA GPU Operator.
+<6> Set this value to `false` to use GPU passthrough with VMs.
```

Comment:
Needs updated to `Set this value to `false` to not load vfio-pci driver needed for PCI Passthrough`

Customers should continue to use [1] - to configure PCI Passthrough

[1] - https://65985--docspreview.netlify.app/openshift-enterprise/latest/virt/virtual_machines/advanced_vm_management/virt-configuring-pci-passthrough